bundles/org.openhab.binding.mqtt.awtrixlight/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,593 @@
+			</channel>
+			<channel id="buttonRight" typeId="system.rawbutton">
+				<label>Button Right</label>
+				<description>Right button pressed</description>
+			</channel>
+			<channel id="buttonSelect" typeId="system.rawbutton">
+				<label>Button Select</label>
+				<description>Select button pressed</description>
+			</channel>
+			<channel id="center" typeId="center"/>
+			<channel id="color" typeId="mainColor"/>
+			<channel id="duration" typeId="duration"/>
+			<channel id="effect" typeId="effect"/>
+			<channel id="effectBlend" typeId="effectBlend"/>
+			<channel id="effectPalette" typeId="effectPalette"/>
+			<channel id="effectSpeed" typeId="effectSpeed"/>
+			<channel id="fadeText" typeId="fadeText"/>
+			<channel id="gradientColor" typeId="gradientColor"/>
+			<channel id="icon" typeId="icon"/>
+			<channel id="lifetime" typeId="lifetime"/>
+			<channel id="lifetimeMode" typeId="lifetimeMode"/>
+			<channel id="line" typeId="line"/>
+			<channel id="overlay" typeId="overlay"/>
+			<channel id="progress" typeId="progress"/>
+			<channel id="progressBackground" typeId="progressBackground"/>
+			<channel id="progressColor" typeId="progressColor"/>
+			<channel id="pushIcon" typeId="pushIcon"/>
+			<channel id="rainbow" typeId="rainbow"/>
+			<channel id="scrollSpeed" typeId="scrollSpeed"/>
+			<channel id="text" typeId="text"/>
+			<channel id="textCase" typeId="textCase"/>
+			<channel id="textOffset" typeId="textOffset"/>
+			<channel id="topText" typeId="topText"/>
+		</channels>
+		<properties>
+			<property name="appid"></property>
+			<property name="thingTypeVersion">2</property>
+		</properties>
+		<representation-property>appid</representation-property>
+		<config-description-ref uri="thing-type:mqtt:awtrixapp"/>
+	</thing-type>
+
+	<channel-type id="lux">
+		<item-type>Number:Illuminance</item-type>
+		<label>Lux</label>
+		<description>Brightness in lux</description>
+		<category>sun</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Light</tag>
+		</tags>
+		<state readOnly="true" pattern="%.0f %unit%"/>
+	</channel-type>
+
+	<channel-type id="autoBrightness">
+		<item-type>Switch</item-type>
+		<label>Automatc Brightness</label>
+		<description>Let the clock set brightness automatically based on internal brightness sensor readings</description>
+		<category>screen</category>
+		<tags>
+			<tag>Control</tag>
+			<tag>Light</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="brightness">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Display Brightness</label>
+		<description>Screen brightness in percent</description>
+		<category>screen</category>
+		<tags>
+			<tag>Control</tag>
+			<tag>Light</tag>
+		</tags>
+		<state pattern="%.0f %unit%"></state>
+	</channel-type>
+
+	<channel-type id="rssi">
+		<item-type>Number:Dimensionless</item-type>
+		<label>WiFi Signal</label>
+		<description>RSSI value of WiFi signal</description>
+		<category>qualityofservice</category>
+		<tags>
+			<tag>Measurement</tag>
```

Comment:
A point (provided) and property are mandatory. As there does not seem to be a good poeprty for RSSI, these tags should be dropped.

Reply:
Done